### DISCUSSION TITLE
Wikipedia article "Comparison_of_text_editors"

### DISCUSSION THREAD

**Discussion by efa** [unixwork/xnedit#80] [Show and tell]
I added XNEdit to the list of editors in the Wiki article:
https://en.wikipedia.org/wiki/Comparison_of_text_editors
I left "Large file support" to no.
I modified the features where I know XNEdit added support over NEdit (like Unicode).
Please check if those features are correct and feel free to complete the modifications

**Comment by unixwork** [comment]
Thanks.

Only one small detail was wrong. XNEdit doesn't support OpenVMS, but I fixed that in the article.
